.travis.yml

```diff
@@ -1,15 +1,14 @@
 language: node_js
 node_js:
+- 10
 - 8
-- 6
 cache:
   directories:
   - "~/.npm"
 notifications:
   email:
     recipients:
-    - arizzitano@edx.org
-    - jbradley@edx.org
+    - abutterworth@edx.org
```

Comment:
we can fairly easily add a google group recipient here if you'd like to not be the only one feeling the spam (should spam arise)

Reply:
Good call I'll make that happen